src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -123,4 +125,18 @@ public void testAnnotation() throws Exception {
             getPath("annotation"
                     + File.separator + "package-info.java"), expected);
     }
+
+    @Test
+    public void testWithFileWithoutParent() throws Exception {
+        final DefaultConfiguration moduleConfig = createModuleConfig(JavadocPackageCheck.class);
+        final File fileWithoutParent = spy(new File(getPath("annotation"
```

Comment:
Why we need `spy` here?

Reply:
Because we use other File methods in
```protected void verify(Checker checker, File[] processedFiles, String messageFileName, String... expected)```
 E.g. we open FileInputStream when creating FileText.

Reply:
@AADudkin Please see this issue, https://github.com/checkstyle/checkstyle/issues/4970 , if you continue to use powermock in this test.

Reply:
Added javadoc for test